storage/pebble/config.go

```diff
@@ -22,16 +22,15 @@ func DefaultPebbleOptions(logger zerolog.Logger, cache *pebble.Cache, comparer *
 
 		// When the maximum number of bytes for a level is exceeded, compaction is requested.
 		LBaseMaxBytes: 64 << 20, // 64 MB
-		Levels:        make([]pebble.LevelOptions, 7),
 		MaxOpenFiles:  16384,
 
 		// Writes are stopped when the sum of the queued memtable sizes exceeds MemTableStopWritesThreshold*MemTableSize.
 		MemTableSize:                64 << 20,
 		MemTableStopWritesThreshold: 4,
 
-		// The default is 1.
-		MaxConcurrentCompactions: func() int { return 4 },
-		Logger:                   util.NewLogger(logger),
+		// The default is 1, 1.
+		CompactionConcurrencyRange: func() (upper int, lower int) { return 1, 4 },
```

Comment:
made into a range: https://github.com/cockroachdb/pebble/pull/4635

should this be `4, 4`?